pages.zh/common/make.md

```diff
@@ -0,0 +1,33 @@
+# make
+
+> Makefile 文件描述目标的任务运行器。
+> 通常用于控制源代码中可执行文件的编译。
+> 更多信息见：<https://www.gnu.org/software/make/manual/make.html>.
+
+- 调用 Makefile 中指定的第一个目标（通常命名为 “all”）：
```

Comment:
Is there a need to use this non-ASCII style of quotes @BillLucky?
```suggestion
- 调用 Makefile 中指定的第一个目标（通常命名为 "all"）：
```

Reply:
I think you are right, it is more natural to use ASCII symbols :) @bl-ue